home-service/src/main/java/com/softserveinc/ita/homeproject/service/exceptions/NotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.softserveinc.ita.homeproject.service.exceptions;
```

Comment:
I don't see any purpose in providing plural package naming. Can we make it singular?

Reply:
Sure we can, thanks